runelite-client/src/main/java/net/runelite/client/chat/ChatCommandManager.java

```diff
@@ -174,4 +176,8 @@ private static String extractCommand(String message)
 
 		return message.substring(0, idx);
 	}
+
+	public List<String> getCommands(){
+		return new ArrayList<>(commands.keySet());
```

Comment:
This looks like it would not have any kind of guarantee for what order the keys are returned. I would think it best for us to sort them, either here or in the chat commands plugin.